DOCS-325 - Service accounts (#5041)
SumoLogic/sumologic-documentation: docs/manage/security/service-accounts.md

Any access keys you add on a service account appear on the [**Access Keys** tab](/docs/manage/security/access-keys/#from-the-access-keys-tab).
:::

## Modify a service account

1. [**Classic UI**](/docs/get-started/sumo-logic-ui-classic). In the main Sumo Logic menu, select **Administration > Security > Service Accounts**. <br/>[**New UI**](/docs/get-started/sumo-logic-ui). In the top menu select **Administration**, and then under **Account Security Settings** select **Service Accounts**. You can also click the **Go To...** menu at the top of the screen and select **Service Accounts**.
1. Hover your mouse over a service account and click the three-dot kebab icon to reveal the modification options.<br/><img src={useBaseUrl('/img/security/modify-options-for-service-accounts.png')} alt="Edit a service account" style={{border: '1px solid gray'}} width="700"/>

:::warning
When a service account is deactivated, the access keys on the service account are also deactivated. For more information about deactivation, see [Access Keys deactivation policy](/docs/manage/security/access-keys/#access-keys-deactivation-policy).
:::

## Edit, deactivate, or delete an access key on a service account

> If instead of modifying a service account itself you want to modify only the access keys on the service account, open the service account, hover your mouse over an access key, and click the three-dot kebab icon to reveal the modification options.

Reviewer: Was this what you meant?

```suggestion
To modify only the access keys on a service account (rather than the service account itself), open the service account, hover over an access key, and click the three-dot kebab icon to reveal modification options.
```

Author: Yes, your wording is great, @kimsauce.
